### PR TITLE
Second big run at cleanup:

### PR DESCRIPTION
**Pull request by plobbes**
- prefer https to http in URLs
- avoid smartquotes and emdashes
- fix typos and invalid examples
- fix some inconsistency of term usage (still more that could be done)
- fix some bogus data from format conversion
- attempt to improve some of the formatting conversion
- be more consistent with terms/verbiage
- take advantage of some table formatting/style capabilities
- MySQL to MariaDB in several places

### CODE REVIEW

File: adminconsole.adoc [hunk 1]
```diff
@@ -21,8 +21,8 @@ Console or the command line.
 
 * *Delegated Administrators* are granted customized administrator roles
 bythe global administrator to manage different tasks from the
```
Comment (fatkudu) [inline adminconsole.adoc]: s/bythe/by the/

File: cmdlineutils.adoc [hunk 2]
```diff
@@ -898,64 +894,64 @@ Modifies the LDAP settings.
 ==== Distribution List Provisioning Commands
 
 .`zmprov` -- Distribution List Provisioning Commands
-[cols=",,a",options="header",]
+[cols="1m,2m,2a",options="header",]
 |=======================================================================
 |Commnad |Syntax |Example/Notes
 
-|`createDistribution List (cdl)` |
+|createDistribution List (cdl) |
```
Comment (fatkudu) [inline cmdlineutils.adoc]: s/createDistribution List/createDistributionList/

File: cos.adoc [hunk 3]
```diff
@@ -45,26 +41,27 @@ in policies.
...
 If a COS is not explicitly set for a new account, or if the COS assigned to
-a user no longer exists, the Zimbra default COS is automatically assigned.
+a user no longer exists, the *default* COS is automatically assigned.
 
-You can create a domain COS and have all accounts that are created on that
-domain automatically assigned to this COS. You can create different COSs
-and specify which ones are available for the domain. If a domain does not
-have a COS defined, and you do not specify a COS, the original default COS
-is automatically assigned when an account is created.
+You can create a COS and and assign that as a default COS for all
```
Comment (fatkudu) [inline cos.adoc]: s/and and/and/

File: delegatedadmin.adoc [hunk 4]
```diff
@@ -97,18 +96,18 @@ Zimlet entry
 == Rights
 
 Rights are the functions that a delegated administrator can or cannot
-perform on a named target. Right can be either system-defined or attribute.
+perform on a named target. A right can be either system-defined or attribute.
```
Comment (fatkudu) [inline delegatedadmin.adoc]: s/attribute/name of an attribute/  ???

File: mta.adoc [hunk 5]
```diff
@@ -840,8 +773,9 @@ Specify a non-zero time value (an integral value plus an optional
 one-letter suffix that specifies the time unit). Time units: s (seconds), m
 (minutes), h (hours), d (days), w (weeks).
 
-Default = postscreen dnsbl ttl. Note that the default seeting is
-backwards-compatible with Postscreen versions earlier than 3.1.
+Note that the default seeting is backwards-compatible with Postscreen
```
Comment (fatkudu) [inline mta.adoc]: s/seeting/setting/

File: provisioning.adoc [hunk 6]
```diff
@@ -50,7 +48,7 @@ Wizard from the Administration Console.  You can import accounts from
 either a generic IMAP server or from another {product-abbrev} server.
 
 [NOTE]
-Only accounts on {product-abbrev} 7.2 or later can be migrated to {product-abbrev} 8.0.
+Only accounts on {product-abbrev} 7.2 or later can be migrated to {product-abbrev} 8.
```
Comment (fatkudu) [inline provisioning.adoc]: 7.2.0 to be consistent with below?

**Comment by fatkudu** [comment]
+1 apart from minor comments above

File: delegatedadmin.adoc (hunk 4 above)
Comment (plobbes) [inline delegatedadmin.adoc]: Proposed change:

Rights are the functions that a delegated administrator can or cannot
perform on a named target. A right is either system-defined or granted
at the attribute level.

File: provisioning.adoc (hunk 6 above)
Comment (plobbes) [inline provisioning.adoc]: Personal preference: 7.2 in both places.

File: delegatedadmin.adoc (hunk 4 above)
Comment (fatkudu) [inline delegatedadmin.adoc]: +1

File: provisioning.adoc (hunk 6 above)
Comment (fatkudu) [inline provisioning.adoc]: +1

**Comment by gordyt** [comment]
+1

**Pull request merged**